site/content/integrate/apps/_index.md

```diff
@@ -5,23 +5,24 @@ description: "Apps are lightweight interactive add-ons to Mattermost."
 weight: 30
 ---
 
-Apps are lightweight, interactive add-ons to Mattermost which can be hosted as HTTP services, or as serverless functions on AWS Lambda, OpenFaaS, Kubernetes, etc. to run without dedicated infrastructure. Apps can:
+Apps are lightweight, interactive add-ons to Mattermost. They are hosted as their own service, as opposed to plugins which run directly alongside the server process.
+
+Apps can:
 
 - Display interactive, dynamic modal forms and message actions.
 - Be written in any language.
-- Attach themselves to locations in the Mattermost user interface (e.g. channel bar buttons, post menu, channel menu, commands), and can add their custom slash commands with full autocomplete.
+- Attach themselves to locations in the Mattermost user interface (e.g. channel bar buttons, post menu, channel menu, commands), and add their custom slash commands with full autocomplete.
 - Receive webhooks from Mattermost, and third-parties, and use the Mattermost REST APIs to post messages, etc.
-- Work on both Mobile and Desktop clients so developers can focus on the functionality of their Apps.
-- Be deployed using our serverless hosting infrastructure keeping data secure and supporting scalability by being stateless.
+- Work on both Mobile and Desktop clients. This way, developers can focus on the features of their Apps without writing code for each specific platform.
 - Include dynamic fields, code as an on-demand function, and interactive modals.
 
-When you develop using the App framework, your Apps can:
+When you develop using the Apps framework, your Apps can:
 
 - Create slash commands
 
   ![image](app-slash-command-zoomed-in.png)
 
-- Post messages to channels
+- Post messages to channels (and anything else possible with [Mattermost's API](https://api.mattermost.com/))
```

Comment:
```suggestion
- Post messages to channels (and anything else possible with {{<newtabref title="Mattermost's API" href="https://api.mattermost.com/">}})
```

This suggestion will make the link open in a new tab. We should do this for any link that takes the user away from the docs site.

Reply:
I like this approach!